app/services/feature-flags.js

```diff
@@ -18,14 +18,14 @@ export default class FeatureFlagsService extends Service {
     return this.currentUser?.isStaff || this.getFeatureFlagValue('can-see-short-instructions-for-stage-2') === 'test';
   }
 
-  get canSeeTweaksForStage1() {
-    return this.currentUser?.isStaff || this.getFeatureFlagValue('can-see-tweaks-for-stage-1') === 'test';
-  }
-
   get currentUser() {
     return this.authenticator.currentUser;
   }
 
+  get hideTestRunnerCardBeforeUserHasSubmitted() {
```

Comment:
```suggestion
  get canSeeForumLinkBeforeStage1Submission() {
```

```suggestion
  get cannotSeeTestRunnerCardBeforeStage1Submission() {
```


@andy1li the suggestions above are (a) most consistent with what we usually name feature flags (b) more specific, since they mention stage 1